### ISSUE TITLE
Security Log events are time-delayed

### ISSUE THREAD

**Issue by kekukui** [wokhansoft/WFN#9]
I am seeing constant activity in the security log that is several minutes old, from applications which have been closed.  This makes it difficult to trap and troubleshoot other traffic blocking issues I want to address. 

**Suggestions:**

1. Implement an option to right click in the list & filter out specific applications
2. Include filter check boxes for "show system processes" & "show user processes"
3. Add a button to purge the security log

4. Always display blocked traffic in real time.  Permit the user to freeze the log and scroll back, instead of waiting for log to catch up with last event.  (See Microsoft "Process Monitor" and "Disk Monitor" (or the revoked "FileMon") from Sysinternals Suite for practical examples of filtering high-traffic logs.

https://technet.microsoft.com/en-us/sysinternals/bb842062.aspx

**Comment by AtlasHackert** [comment]
(There's a bug in the security log feature, where WFN keeps finding old entries over and over again, leading to a lot of duplication. This wil be fixed in V2.0 BETA 3.)

**Comment by AtlasHackert** [comment]
Can you check if this issue is still present in the just released v2.0 Beta 3? I've made some improvements to the security log loading.

**Comment by h3298** [comment]
It appears the issue is resolved.  I spent a lot of time testing it by using the browser and watching the log.  Now that the duplicate entries are out of the way and the log is more stable, I noticed something interesting:

The Chrome browser constantly spams my LAN with multicast traffic (239.255.255.250, 224.0.0.251) looking for other "things".  It also tries to bypass my router's firewall by asking it for UPnP services when they are not needed:

**Unexpected network traffic showing in Chrome Task Manager**
https://superuser.com/questions/678468/unexpected-network-traffic-showing-in-chrome-task-manager

**How to disable Google Chrome's unrequested connections?**
https://stackoverflow.com/questions/5814334/how-to-disable-google-chromes-unrequested-connections

The Chrome browser apparently ignores system DNS settings too: instead of obeying DNS preferences on the local machine and the router, it uses built-in Google DNS, so Google can keep a history of your surfing and match your IP with their cookies.  (I cant think of any other reason for this.)

It appears that Chrome will also defeat any DNSSEC or DNScrypt security features on your router by making insecure, unencrypted DNS requests on port 80 via 'Host Process for Windows Services'.  If you block port 80, it will try on 443.  (If you just sit there doing nothing and WFN is set to "block and prompt" for everything, you will periodically see this background activity.)

It occurs to me that Microsoft omitted this notification capability from the Windows firewall because a lot of Microsoft apps also make a fair amount of unnecessary connections.  I also found the map in WFN kind of interesting.  Maybe someday Etherape will get ported to Windows -- that would be even better!

https://www.youtube.com/watch?v=oPlnB-Lu3vs
